woocommerce/wc-admin
# Add coupons count to data store and hook up to summary numbers in orders report

Fixes #1695 

~~Adds currency formatting to the coupons summary.~~

Changes the coupon numbers in the table to use distinct coupon code count instead of the total coupon amount.

### Detailed test instructions:

~~1. Go to the Orders report.
2. Check that the coupons summary number is formatted as a currency.~~

1.  Add some orders with coupons.  Try adding multiple coupons to an order and using the same coupon across different orders.
2.  Check that the Orders report coupon number reflects the unique number of coupons during a selected period.
3.  Check that all unit tests are passing.
